src/main/java/io/buildpacks/example/sample/GreetingController.java

```diff
@@ -7,8 +7,14 @@
 @RestController
 public class GreetingController {
 
```

Comment:
3. Consider adding error handling and proper exception messages for potential exceptions that may be thrown in the `greeting` method, especially when dealing with user input.